### DISCUSSION TITLE
Configure before connecting to A/C?

### DISCUSSION THREAD

**Discussion by ThiefMaster** [revk/xESP32-Faikout#105] [General]
Is it possible to just connect the power lines in order to setup the wifi data etc. before connecting it to the actual A/C?

**Comment by revk** [comment]
No problem

**Reply by ThiefMaster** [reply]
Do I need to connect the 5V line, the 12V line or (hopefully not!) both?

**Reply by revk** [reply]
It runs off the 12V. The 5V is just a reference for data.
